src/containers/shared/components/Transaction/OracleSet/test/ConvertScalePrice.test.js

```diff
@@ -0,0 +1,26 @@
+import { convertScaledPrice } from '../parser'
+
+describe('convertScaledPrice', () => {
```

Comment:
nit: should use TX (`ConvertScalePrice.test.ts`)

Reply:
Could you clarify a bit here?

Reply:
Typo - I meant TS (TypeScript)